src/main.rs

```diff
@@ -50,6 +51,17 @@ enum Opt {
         )]
         sort: SortOrder,
 
+        /// Analyze .ll files that were produced by eg. `RUSTFLAGS="--emit=llvm-ir" ./x.py build --stage 0 compiler/rustc`.
```

Comment:
This should mention that it ignores files that don't end in `.ll`, I would not expect that behavior. Maybe it shouldn't ignore them?

Reply:
Changed in #33 to not ignore any files.